core/src/main/kotlin/com/github/zeldigas/text2confl/core/config/ConverterConfig.kt

```diff
@@ -8,15 +8,15 @@ import io.ktor.http.*
 import java.nio.file.Path
 
 data class ConverterConfig(
-    val titlePrefix: String,
-    val titlePostfix: String,
-    val editorVersion: EditorVersion,
-    val addAutogeneratedNote: Boolean?,
-    val docsLocation: String?,
-    val autogeneratedNote: String?,
-    val codeBlockParams: CodeBlockParams,
-    val markdownConfig: MarkdownConfiguration,
-    val asciidocConfig: AsciidoctorConfiguration
+        val titlePrefix: String,
```

Comment:
it looks like your idea has different formatting settings, and it produces changes that are not relevant at all.

not to myself - store in project IDEA settings to avoid such issues for future contributions